### PR TITLE
Add test for svelte and improve error message when plugin is missing

### PR DESCRIPTION
**Pull request by calebeby**
@Paul-Hebert asked whether Pleasantest supported Svelte and I realized I hadn't actually tested it out. This PR adds a test to make sure it works (it did, without changes).

I also added a suggestion for error messages when there are syntax errors while parsing a non-JS file that is imported into JS (i.e. it will suggest adding a plugin for parsing .vue or .svelte files if you import them and there is a parse error after all user plugins have run)

### CODE REVIEW

File: tests/utils/svelte-component.svelte [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	Clicked {count} {count === 1 ? 'time' : 'times'}
+</button>
+
+<style lang="scss">
+$bg: red;
+
+button {
+  background: $bg;
+}
+</style>
```
Comment (gerardo-rodriguez) [inline tests/utils/svelte-component.svelte:20]: Nit, can you adjust the tabbing in this file, it's all over the place. 😅

**Review by gerardo-rodriguez — APPROVED** [review]
One nit comment, otherwise, LGTM! 👍

File: tests/utils/svelte-component.svelte (hunk 1 above)
Comment (calebeby) [reply tests/utils/svelte-component.svelte:20]: Oh yeah good point, this is what I get for copy-pasting svelte examples from a bunch of places 😂

**Comment by Paul-Hebert** [comment]
This is awesome, thanks @calebeby !

**Pull request merged**